.github/workflows/update-google-fons-data.yml

```
@@ -0,0 +1,44 @@
+# This is a basic workflow that is manually triggered
+
+name: Update Google Fonts JSON
+
+# Controls when the action will run. Workflow runs when manually triggered using the UI
+# or API.
+on:
+  schedule:
+    - cron: '0 0 1,7,14,21,28 * *' # Runs weekly, more precisely at 00:00 on day-of-month 1, 7, 14, 21, and 28.
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "update-google-fonts-json"
+  update-google-fonts-json:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 18
+
+    - name: Get current date
+      id: date
+      run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
+
+    # Runs a single command using the runners shell
+    # This script fetchs the Goolgle Fonts API data and creates a PR if new data is available
+    - name: Update Google Fonts JSON
+      env:
+        GOOGLE_FONTS_API_KEY: ${{ secrets.GOOGLE_FONTS_API_KEY }}
+        BRANCH_NAME: update/google-fonts-json-${{ steps.date.outputs.date }}_${{ github.run_id }}
+        GITHUB_TOKEN: ${{ secrets.REPO_TOKEN }}
+      run: |
+        echo "Updating Google fonts JSON"
+        node ./update-google-fonts-json-file.js
+        git config user.name 'Matias Benedetto'
+        git config user.email 'matias.benedetto@gmail.com'
```

```suggestion
        git config user.name 'github-actions[bot]'
        git config user.email 'github-actions[bot]@users.noreply.github.com'
```

I'd perhaps suggest changing the author of these PRs to the GH actions bot.

---

Amazing! :stars:  I didn't know about the gh bot user.